scripts/README.md

```diff
@@ -1,75 +1,45 @@
-NETWORK=ethereum-sepolia-testnet DAPI_NAME=BTC/USD OEV_BENEFICIARY=0x55Cf1079a115029a879ec3A11Ba5D453272eb61D yarn deploy-dapi-proxy-with-oev
+NETWORK=ethereum-sepolia-testnet PROXY=0x3f16e1a6b915C42f6442bF59b092FaBb7d5eBB82 yarn update-proxy
 ```
 
-To deploy a dAPI proxy programmatically, see the command below, but use your own `NETWORK` and `DAPI_NAME` values.
+- Use the command below to confirm that `DataFeedReaderExample` now reads the new data feed
 
 ```sh
-NETWORK=ethereum-sepolia-testnet DAPI_NAME=BTC/USD yarn deploy-dapi-proxy
+NETWORK=ethereum-sepolia-testnet yarn read-data-feed
 ```
 
-You can see the valid dAPI names on API3 Market.
-
-### DataFeedProxyWithOev and DataFeedProxy
-
-> **WARNING:** DapiProxy reads a data feed whose composition is managed by API3, and using it is essentially outsourcing data feed curation to API3.
-> DataFeedProxy reads a data feed whose composition is immutable, which is fully trust-minimized when used with first-party oracles.
-> However, this also means when there is something wrong with the composition of the DataFeedProxy you are using, you will need to deploy a new one and switch to that, i.e., you will need to do your own data feed curation.
-> Do not prefer DataFeedProxy over DapiProxy unless you understand the distinction and are capable of managing your own data feeds.
+## Deploying proxy contracts programmatically
 
-See the command below, but use your own `NETWORK` and `DATA_FEED_ID` values. The OEV beneficiary address defauls to a gnosis safe multisig address managed by API3, which in this example is `0x55Cf1079a115029a879ec3A11Ba5D453272eb61D`.
+The [instructions](../README.md#instructions) have you use a `DapiProxyWithOev` contract that gets deployed automatically with a subscription purchase.
+To deploy the same proxy programmatically, use the command below with your `NETWORK` and `PROXY` values.
 
 ```sh
-NETWORK=ethereum-sepolia-testnet DATA_FEED_ID=0xd888b92f9d71afedd0a012622c0d1d5368fc0dc0ff1d30bb16266afcd49c2c17 yarn deploy-data-feed-proxy-with-oev
+NETWORK=ethereum-sepolia-testnet DAPI_NAME=ETH/USD yarn deploy-dapi-proxy-with-oev
 ```
 
-If you'd like to set your own OEV beneficiary address then use the `OEV_BENEFICIARY` environment variable.
+You can also refer to the [`deploy-dapi-proxy-with-oev` script](./deploy-dapi-proxy-with-oev.js) to see how the `@api3/contracts` package can be used to derive the respective `DapiProxyWithOev` address without needing to make any RPC calls.
 
-```sh
-NETWORK=ethereum-sepolia-testnet DATA_FEED_ID=0xd888b92f9d71afedd0a012622c0d1d5368fc0dc0ff1d30bb16266afcd49c2c17 OEV_BENEFICIARY=0x55Cf1079a115029a879ec3A11Ba5D453272eb61D yarn deploy-data-feed-proxy-with-oev
-```
+The OEV beneficiary of the proxy above is the API3 _manager multisig_ on that chain.
+To deploy a proxy with an arbitrary OEV beneficiary, use the command below with your `OEV_BENEFICIARY` value.
+Note that the OEV functionalities of proxies with custom beneficiaries are not supported
```

Comment:
For someone just trying to read a data feed value all this OEV stuff might be a bit overwhelming. I wonder if we should add a link to some link in the docs. [This one](https://docs.api3.org/explore/dapis/OEV.html) for instance

Reply:
It's in the advanced section so it should be fine. I don't want to link OEV docs until they reach a stable state.